### ISSUE TITLE
Disable printlns

### ISSUE THREAD

**Issue by bootstraponline** [linkedin/dex-test-parser#10]
```java
    println("Found $count fully qualified test methods")
            }
            println("Finished in $time ms")
```

These printlns always execute. There should be a way to toggle them off.

**Comment by drewhannay** [comment]
@kkoser can you fix this with your PR for #9? Might be easiest to just remove them...I only had these lines for my own debugging while developing.

**Comment by kkoser** [comment]
Removed these logs in #16
